src/background/services/globalConfigService.ts

```diff
@@ -65,9 +68,9 @@ export default class GlobalConfigService {
       if (!config.myDapplets) config.myDapplets = this.getInitialConfig().myDapplets
       if (!config.targetStorages) config.targetStorages = this.getInitialConfig().targetStorages
       if (!config.xdaiProviderUrl) config.xdaiProviderUrl = this.getInitialConfig().xdaiProviderUrl
-      if (!config.connectedAccountsContractAddress)
-        config.connectedAccountsContractAddress =
-          this.getInitialConfig().connectedAccountsContractAddress
+      if (!config.preferredConnectedAccountsNetwork)
+        config.preferredConnectedAccountsNetwork =
+          this.getInitialConfig().preferredConnectedAccountsNetwork
```

Comment:
It seems that you forgot 'connectedAccountsTestnetContractAddress', 'connectedAccountsMainnetContractAddress'.

Reply:
https://github.com/dapplets/dapplet-extension/commit/a95011495cc4ac2270ce8bc261e097f2ceb415f1
fixed